heidegger_index/views.py

```diff
@@ -79,6 +78,37 @@ class LemmaDetailView(DetailView):
     template_name = "lemma_detail.html"
     context_object_name = "lemma"
 
+    def get_object(self, queryset=None):
+        """
+        CRUDE COPY OF get_object()
+        """
+        if queryset is None:
+            queryset = self.get_queryset()
+
+        slug = self.kwargs.get(self.slug_url_kwarg)
+        gnd = self.kwargs.get('gnd')
+        urn = self.kwargs.get('urn')
+        if slug is not None:
+            slug_field = self.get_slug_field()
+            queryset = queryset.filter(**{slug_field: slug})
+        if gnd is not None and slug is None:
+            queryset = queryset.filter(gnd=gnd)
+        if urn is not None and slug is None and gnd is None:
+            queryset = queryset.filter(urn=urn)
+        # If none of those are defined, it's an error.
+        if slug is None and gnd is None and urn is None:
+            raise AttributeError(
+                "Generic detail view %s must be called with either an object "
+                "slug, urn or gnd in the URLconf." % self.__class__.__name__
+            )
+        try:
+            # Get the single item from the filtered queryset
+            obj = queryset.get()
+        except queryset.model.DoesNotExist:
+            raise Http404(_("No %(verbose_name)s found matching the query") %
+                        {'verbose_name': queryset.model._meta.verbose_name})
+        return obj
+
```

Comment:
Hier een ander voorstel voor het laden van het juiste lemma a.d.h.v. de `gnd` of `urn` in de url. Ik weet nou niet of dit nu veel netter is.